cmd/prober/prober.go

```diff
@@ -201,6 +221,15 @@ func runProbers(ctx context.Context, freq int, runOnce bool) {
 				Logger.Errorf("error running request %s: %v", r.Endpoint, err)
 			}
 		}
+
+		// Performing requests against Fulcio gRPC API
+		for _, method := range fulciopb.CA_ServiceDesc.Methods {
```

Comment:
Rather than iterate over all possible methods, can we just pick one representative endpoint? `GetTrustBundle`?

Reply:
Sure!! I tried to replicate the behavior for http requests, making requests for all available endpoints.